### ISSUE TITLE
Pick N cards is not fully supported

### ISSUE THREAD

**Issue by Mate119** [Pandapip1/SelectAnyNumberRounds#1]
If you choose to take multipule decks in 1 round the mod will bug out and choose the cards from the second deck without showing you what the cards are

**Comment by NameLord** [comment]
Mod abandoned wow....

**Comment by Pandapip1** [comment]
Yup, has been for a while. I might consider merging the code into Pick N Cards itself so it's at least somewhat maintained.
